aQute.libg/src/aQute/lib/io/packageinfo

```diff
@@ -1 +1 @@
-version 3.2
+version 3.2.1
```

Comment:
This should be `3.3`. You are adding a new method. 

Because we have not released 4.2, we are still baselining 4.3 against 4.1. This will be changed to baseline against 4.2 once it is released and then not making it `3.3` will be an error.